<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentHome</class>
 <widget class="QMainWindow" name="StudentHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main window background */
QMainWindow {
    background-color: #f0f0f0; /* Светло-серый фон */
}

/* General button style */
QPushButton {
    background-color: #4CAF50; /* Зеленый фон */
    border: none; /* Без границы */
    color: white; /* Белый текст */
    padding: 10px 24px; /* Внутренний отступ */
    text-align: center; /* Выравнивание текста по центру */
    font-size: 16px; /* Размер шрифта */
    margin: 4px 2px; /* Внешний отступ */
    border-radius: 12px; /* Закругленные углы */
}

QPushButton:hover {
    background-color: #45a049; /* Темно-зеленый фон при наведении */
}

QPushButton:pressed {
    background-color: #398439; /* Ещё темнее при нажатии */
}

/* Line edit fields */
QLineEdit {
    border: 1px solid #ccc; /* Светло-серая граница */
    border-radius: 10px; /* Закругленные углы */
    padding: 10px; /* Внутренний отступ */
    font-size: 16px; /* Размер шрифта */
    background-color: #fff; /* Белый фон */
    margin: 4px 0; /* Внешний отступ */
}

/* Line edit focus style */
QLineEdit:focus {
    border: 1px solid #4CAF50; /* Зеленая граница при фокусе */
}

/* Labels */
QLabel {
    font-size: 16px; /* Размер шрифта */
    color: #333; /* Темно-серый текст */
    margin: 4px 0; /* Внешний отступ */
}

/* Combobox */
QComboBox {
    border: 1px solid #ccc; /* Светло-серая граница */
    border-radius: 10px; /* Закругленные углы */
    padding: 8px; /* Внутренний отступ */
    font-size: 16px; /* Размер шрифта */
    background-color: #fff; /* Белый фон */
    margin: 4px 0; /* Внешний отступ */
}

/* Combobox focus style */
QComboBox:focus {
    border: 1px solid #4CAF50; /* Зеленая граница при фокусе */
}

/* Message box buttons */
QMessageBox QPushButton {
    background-color: #4CAF50; /* Зеленый фон */
    color: white; /* Белый текст */
    border: none; /* Без границы */
    padding: 10px 20px; /* Внутренний отступ */
    border-radius: 5px; /* Закругленные углы */
}

QMessageBox QPushButton:hover {
    background-color: #45a049; /* Темно-зеленый фон при наведении */
}

QMessageBox QPushButton:pressed {
    background-color: #398439; /* Ещё темнее при нажатии */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Записаться на факультатив</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выберите факультатив для записи:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>0</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Информация о преподавателе:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Информация о видах проведения:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Zapisatsya">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>230</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Записаться</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InfoPrepodavatelNameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InfoPrepodavatelSurnameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InfoPrepodavatelPatronumicEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Факультатив</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxRegisterFacultativ"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Практики</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InfoConductPracticEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Лекции</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InfoConductLekctiaEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Лабараторные работы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="InfoConductLabEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Перезаписаться на факультатив</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выберите факультатив:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Информация о новом факультативе:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Perezapisatsya">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>310</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Перезаписаться</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Факультатив</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxOldFaciulttativ"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>328</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Факультатив</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxNewFacultativ">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>52</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Практики</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="InfoPerezapisPracEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="minimumSize">
              <size>
               <width>76</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Лекции</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="InfoPerezapisLekctiaEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Лабараторные работы</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="InfoPerezapisLabEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Просмотреть статистику</string>
     </attribute>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Расписание:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Успеваемость:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>251</width>
        <height>115</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Факультатив</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxYspevaemost"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Оценка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="GradeEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>52</y>
        <width>251</width>
        <height>173</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="minimumSize">
            <size>
             <width>124</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Факультатив</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxRaspisanieFacultativov"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Вид проведения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxConductFacultativ">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Время</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TimeEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Профиль</string>
     </attribute>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Профиль:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_28">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_29">
           <property name="minimumSize">
            <size>
             <width>74</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SurnameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <item>
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PatronumicEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="minimumSize">
            <size>
             <width>68</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Группа</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="GroupEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="minimumSize">
            <size>
             <width>68</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Адресс</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="AdressEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Телефон</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PhoneEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти из сессии</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
